mw-webapp/src/component/checkbox/Сheckbox.tsx

```diff
@@ -45,13 +45,18 @@ export const Checkbox = (props: CheckboxProps) => {
   };
 
   return (
-    <input
-      className={clsx(styles.checkbox, props.className)}
-      type="checkbox"
-      checked={isChecked}
-      onChange={() => {
-        props.isEditable !== false && handleCheckboxChange();
-      }}
-    />
+    <>
+      <label htmlFor="checkbox">
+        <input
+          className={clsx(props.className, styles.checkbox)}
+          type="checkbox"
+          id="checkbox"
```

Comment:
Check please is it possible to get rid from id in input and htmlFor in label by using input inside label

Reply:
Done